Algebraic_kernel_d/examples/Algebraic_kernel_d/CMakeLists.txt

```
@@ -5,23 +5,22 @@ find_package(CGAL REQUIRED COMPONENTS Core)
 
 find_package(MPFI QUIET)
 
-if(MPFI_FOUND)
+if(MPFI_FOUND CGAL_Core_FOUND AND MPFI_FOUND)
```

You should try your commits. That one is probably a syntax error.
```suggestion
if(MPFI_FOUND CGAL_Core_FOUND AND MPFI_FOUND)
```
```suggestion
if(CGAL_Core_FOUND AND MPFI_FOUND)
```

---

And you should let me at least 5 minutes when I do a commit to check everything

---

You have time for that *before* the commit, and *before* the push. When you push, that means it goes public, for everybody to review.